modules/nf-core/gt/ltrharvest/main.nf

```diff
@@ -0,0 +1,57 @@
+process GT_LTRHARVEST {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/genometools-genometools:1.6.5--py310h3db02ab_0':
+        'biocontainers/genometools-genometools:1.6.5--py310h3db02ab_0' }"
+
+    input:
+    tuple val(meta), path(index)
+
+    output:
+    tuple val(meta), path("*.tabout")       , emit: tabout      , optional: true
+    tuple val(meta), path("*.gff3")         , emit: gff3        , optional: true
+    tuple val(meta), path("out.fasta")      , emit: fasta       , optional: true    // When args has -out out.fasta
+    tuple val(meta), path("inner.fasta")    , emit: inner_fasta , optional: true    // When args has -outinner inner.fasta
```

Comment:
Are these the default names, or can they be named using `ext.args`? If they can be named using `ext.args`, then this will not work.

Reply:
Thank you @mahesh-panchal 

Really good point. I had made the assumption that the user will always specify the names as out.fasta and inner.fasta in `ext.args`. Now I have removed that assumption and added multiple tests to verify the generalised code. There might be some edge case here that I am missing.